remo/reports/models.py

```diff
@@ -186,6 +189,14 @@ def save(self, *args, **kwargs):
         # Save the mentor of the user if no mentor is defined.
         if not self.mentor:
             self.mentor = self.user.userprofile.mentor
+        # Save the country if possible
+        if not self.country:
+            try:
+                country = self.location.split()[-1].replace(' ', '')
```

Comment:
Use ``str.strip()`` instead of ``replace(' ', '')`` Some country names include a space eg. ``United States``.